internal/cli/server_run.go

```diff
@@ -374,7 +375,9 @@ This command will bootstrap the server and setup a CLI context.
 			telemetryOptions = append(telemetryOptions, telemetry.WithDatadogExporter(
 				datadog.Options{
 					TraceAddr: c.flagTelemetryDatadogTraceAddr,
+					StatsAddr: c.flagTelemetryDatadogTraceAddr,
```

Comment:
Here I simply re-use the current trace address, but we could easily add a separate flag for stats.